p2p_wallet/Coordinators/TabBar/TabBarCoordinator.swift

```diff
@@ -162,9 +162,10 @@ final class TabBarCoordinator: Coordinator<Void> {
         tabBarController.middleButtonClicked
             .receive(on: RunLoop.main)
             // vibration
-            .handleEvents(receiveOutput: {
+            .handleEvents(receiveOutput: { [unowned self] in
                 let generator = UIImpactFeedbackGenerator(style: .light)
                 generator.impactOccurred()
+                analyticsManager.log(event: AmplitudeEvent.actionButtonClick(isSellEnabled: sellDataService.isAvailable))
```

Comment:
why does event name starts with Amplitude?
Is it a general event or only for amplitude?

Reply:
only amplitude. After removing our old AnalyticsManager it's going to be as .actionButtonClick without AmplitudeEvent